source/wp-content/themes/wporg-developer-2023/js/function-reference.js

```diff
@@ -60,16 +64,20 @@ jQuery( function ( $ ) {
 		const $container = $( document.createElement( 'div' ) );
 		$container.addClass( 'wp-code-block-button-container' );
 
-		$container.append( $copyButton );
+		$container.append( '<code>' + wporgFunctionReferenceI18n.sourceFile + '</code>' );
+
+		const $btnContainer = $( document.createElement( 'span' ) );
 
 		// Check code block height, and if it's larger, add in the collapse
 		// button, and set it to be collapsed differently.
 		const originalHeight = $element.height();
 		if ( originalHeight > MIN_HEIGHT ) {
 			$element.data( 'height', originalHeight );
 
-			const $expandButton = $( document.createElement( 'button' ) );
-			$expandButton.on( 'click', function () {
+			const $expandButtonContainer = $( '<div class="wp-block-button is-style-outline"></div>' );
+			const $expandButton = $( '<a href="#" class="wp-block-button__link wp-element-button"></a>' );
```

Comment:
Seems like it would be a good a11y improvement to also add an `aria-controls` attr to this, linking it to the source code wrapper, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded